lib/poker_hands/hand_rankings/pair.rb

```diff
@@ -1,5 +1,11 @@
 class Pair
   def check(hand)
-    true
+    hand.sort_by! { |card| card.rank.to_i }
```

Comment:
Do we necessarily need to sort the hand in order to find pairs?

Reply:
Not at all, removed in the second commit.